src/lib/language.js

```diff
@@ -0,0 +1,55 @@
+import store from '../store';
+
+/**
+ * To normalize Language String and return language code
+ * @param {String} languageString
+ */
+const normalizeLanguageString = (languageString) => {
+	let [languageCode, countryCode] = languageString.split ? languageString.split(/[-_]/) : [];
+	if (!languageCode || languageCode.length !== 2) {
+		return 'en';
+	}
+	languageCode = languageCode.toLowerCase();
+
+	if (!countryCode || countryCode.length !== 2) {
+		countryCode = null;
+	} else {
+		countryCode = countryCode.toUpperCase();
+	}
+
+	return countryCode ? `${ languageCode }_${ countryCode }` : languageCode;
+};
+
+/**
+ * To get browser Language of user
+ */
+const browserLanguage = () => {
+	navigator.userLanguage || navigator.language;
+};
+
+/**
+ * This is configured langauge
+ */
+const configLanguage = () => {
+	const { config: { settings: { language } = {} } = {} } = store.state;
+	return language;
+};
+
+/**
+ * This will update langauge of widget
+ */
+export const updateWidgetLanguage = () => {
+	const { iframe: { language } = {} } = store.state;
+	I18n.changeLocale(language ? language : normalizeLanguageString(configLanguage() || browserLanguage()));
```

Comment:
```suggestion
	I18n.changeLocale(normalizeLanguageString(configLanguage() || browserLanguage()));
```

Related to the comment I left above..